pages/linux/burpsuite.md

```diff
@@ -0,0 +1,25 @@
+# burpsuite
+
+> A GUI based application that's used in web application penetration testing.
+> CLI is used to control the startup behavior of Burpsuite application.
```

Comment:
What else should it do?

```suggestion
```

Reply:
@pixelcmtd 

Well, the CLI is only used to control startup behavior. So I don't think it's necessary to add more here. So I committed the suggested change there.